src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -1107,6 +1107,14 @@ describe('ReactDOMComponent', function() {
         'Warning: This browser doesn\'t support the `onScroll` event'
       );
     });
+
+    it('should not warn when server-side rendering `onScroll`', function() {
+      spyOn(console, 'error');
+      ReactDOMServer.renderToString(React.createElement('div', {
```

Comment:
I did verify that this test fails without my code change. Mocking `isEventSupported` isn't required because it just returns early no matter what if we're using SSR, so the `warning` call is never reached.